tutorials/RichDisplay.md

```diff
@@ -0,0 +1,91 @@
+[`RichDisplay`](https://klasa.js.org/RichDisplay.html) allows you to create a paginated menu that users of your bot will be able to browse by using reaction-based navigation.
+
+The most simple working example can achieved by this code:
+
+```javascript
+const { Command, RichDisplay } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, { description: 'Test RichDisplay' });
+	}
+
+	async run(msg) {
+		return new RichDisplay()
+			.addPage(new Embed())
+			.run(await msg.send('Loading...'));
+	}
+
+};
+```
+
+A more complex example with an actual usage is this:
+
+```javascript
+const display = new RichDisplay(new this.client.methods.Embed()
+	.setColor(0x673AB7)
+	.setAuthor(this.client.user.name, this.client.user.avatarURL())
+	.setTitle('Random Image Slideshow')
+	.setDescription('Scroll between the images using the provided reaction emotes.')
+);
+
+for (let i = 0; i < 10; i++)
+	display.addPage(template => template.setImage('https://lorempixel.com/400/200/'));
+
+return display.run(await msg.send('Loading slideshow...'));
+```
+
+> The code is contained in the block of the aforementioned command, inside the `async run(msg)` method.
+
+## Code Analysis
+
+First we create a new [`RichDisplay`](https://klasa.js.org/RichDisplay.html) instance, but this time we pass in a [`MessageEmbed`](https://discord.js.org/#/docs/main/master/class/MessageEmbed) instance, which will represent our template, from which we will be able to extend upon to create our pages later on:
+
+```javascript
+const display = new RichDisplay(new this.client.methods.Embed()
+	/* ... */
+);
+```
+
+This [`MessageEmbed`](https://discord.js.org/#/docs/main/master/class/MessageEmbed) instance will be accessible to us by either calling the [`template`](https://klasa.js.org/RichDisplay.html#template__anchor) property of the [`RichDisplay`](https://klasa.js.org/RichDisplay.html) instance, in a cloned manner, or directly through the [`addPage`](https://klasa.js.org/RichDisplay.html#addPage__anchor) method, if we pass in an arrow function or a callback.
+
+```javascript
+for (/* ... */)
+	display.addPage(template => /* ... */);
+```
+
+From here we will be able to add content or edit properties of the template, and then, return the final embed to the [`addPage`](https://klasa.js.org/RichDisplay.html#addPage__anchor) method to be processed into a [`RichDisplay`](https://klasa.js.org/RichDisplay.html) page.
+
+```javascript
+/* ... */
+	display.addPage(template => {
```

Comment:
Unexpected `{`

Reply:
Nice one, I didn't see the curly brace wasn't closing (if so, it must have the semi after setColor aswell).